### PR TITLE
Apiserver spdy

### PR DESCRIPTION
**Pull request by sjanota**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

**TODO**
- [x] Enable authentication and authorisation
- [x] Ensure that no goroutines are leaking
- [x] Cleanup the code
- [x] Check if unit tests make sense and add them if so
- [x] Write acceptance tests for new behaviour
- [x] Ensure that `port-forward` is working

### CODE REVIEW

**Comment by sjanota** [comment]
I've executed some helm commands trough `apiserver-proxy` and they worked fine. It seems that portforward is working as expected.

**Comment by sjanota** [comment]
1. Bare metal
Kubeconfig:
```

apiVersion: v1
clusters:
- cluster:
    certificate-authority: /Users/i355395/.minikube/ca.crt
    server: http://0.0.0.0:8444
  name: minikube
contexts:
- context:
    cluster: minikube
    user: minikube
  name: kyma.local
current-context: kyma.local
kind: Config
preferences: {}
users:
- name: minikube
  user:
    client-certificate: /Users/i355395/.minikube/client.crt
    client-key: /Users/i355395/.minikube/client.key
```
Exec:
```
$ kubectl exec -n kube-system etcd-minikube /bin/sh -it -v10
I0220 13:25:29.353161    6198 loader.go:359] Config loaded from file local-noauth
I0220 13:25:29.353782    6198 loader.go:359] Config loaded from file local-noauth
I0220 13:25:29.354333    6198 loader.go:359] Config loaded from file local-noauth
I0220 13:25:29.354860    6198 round_trippers.go:386] curl -k -v -XGET  -H "Accept: application/json, */*" -H "User-Agent: kubectl/v1.11.0 (darwin/amd64) kubernetes/91e7b4f" 'http://0.0.0.0:8444/api/v1/namespaces/kube-system/pods/etcd-minikube'
I0220 13:25:46.874538    6198 round_trippers.go:405] GET http://0.0.0.0:8444/api/v1/namespaces/kube-system/pods/etcd-minikube 200 OK in 17519 milliseconds
I0220 13:25:46.874554    6198 round_trippers.go:411] Response Headers:
I0220 13:25:46.874559    6198 round_trippers.go:414]     Content-Length: 3530
I0220 13:25:46.874563    6198 round_trippers.go:414]     Content-Type: application/json
I0220 13:25:46.874567    6198 round_trippers.go:414]     Date: Wed, 20 Feb 2019 12:12:43 GMT
I0220 13:25:46.874652    6198 request.go:897] Response Body: {"kind":"Pod","apiVersion":"v1","metadata":{"name":"etcd-minikube","namespace":"kube-system","selfLink":"/api/v1/namespaces/kube-system/pods/etcd-minikube","uid":"efca9d07-34e8-11e9-979d-2e2f58705dfb","resourceVersion":"505","creationTimestamp":"2019-02-20T08:24:27Z","labels":{"component":"etcd","tier":"control-plane"},"annotations":{"kubernetes.io/config.hash":"8eebc7054eafdda97dee9e915b0e67cf","kubernetes.io/config.mirror":"8eebc7054eafdda97dee9e915b0e67cf","kubernetes.io/config.seen":"2019-02-20T08:23:15.127801274Z","kubernetes.io/config.source":"file","scheduler.alpha.kubernetes.io/critical-pod":""}},"spec":{"volumes":[{"name":"etcd-data","hostPath":{"path":"/data/minikube","type":"DirectoryOrCreate"}},{"name":"etcd-certs","hostPath":{"path":"/var/lib/minikube/certs//etcd","type":"DirectoryOrCreate"}}],"containers":[{"name":"etcd","image":"k8s.gcr.io/etcd-amd64:3.2.18","command":["etcd","--advertise-client-urls=https://127.0.0.1:2379","--cert-file=/var/lib/minikube/certs/etcd/server.crt","--client-cert-auth=true","--data-dir=/data/minikube","--initial-advertise-peer-urls=https://127.0.0.1:2380","--initial-cluster=minikube=https://127.0.0.1:2380","--key-file=/var/lib/minikube/certs/etcd/server.key","--listen-client-urls=https://127.0.0.1:2379","--listen-peer-urls=https://127.0.0.1:2380","--name=minikube","--peer-cert-file=/var/lib/minikube/certs/etcd/peer.crt","--peer-client-cert-auth=true","--peer-key-file=/var/lib/minikube/certs/etcd/peer.key","--peer-trusted-ca-file=/var/lib/minikube/certs/etcd/ca.crt","--snapshot-count=10000","--trusted-ca-file=/var/lib/minikube/certs/etcd/ca.crt"],"resources":{},"volumeMounts":[{"name":"etcd-data","mountPath":"/data/minikube"},{"name":"etcd-certs","mountPath":"/var/lib/minikube/certs//etcd"}],"livenessProbe":{"exec":{"command":["/bin/sh","-ec","ETCDCTL_API=3 etcdctl --endpoints=https://[127.0.0.1]:2379 --cacert=/var/lib/minikube/certs//etcd/ca.crt --cert=/var/lib/minikube/certs//etcd/healthcheck-client.crt --key=/var/lib/minikube/certs//etcd/healthcheck-client.key get foo"]},"initialDelaySeconds":15,"timeoutSeconds":15,"periodSeconds":10,"successThreshold":1,"failureThreshold":8},"terminationMessagePath":"/dev/termination-log","terminationMessagePolicy":"File","imagePullPolicy":"IfNotPresent"}],"restartPolicy":"Always","terminationGracePeriodSeconds":30,"dnsPolicy":"ClusterFirst","nodeName":"minikube","hostNetwork":true,"securityContext":{},"schedulerName":"default-scheduler","tolerations":[{"operator":"Exists","effect":"NoExecute"}],"priorityClassName":"system-cluster-critical","priority":2000000000},"status":{"phase":"Running","conditions":[{"type":"Initialized","status":"True","lastProbeTime":null,"lastTransitionTime":"2019-02-20T08:23:15Z"},{"type":"Ready","status":"True","lastProbeTime":null,"lastTransitionTime":"2019-02-20T08:23:16Z"},{"type":"ContainersReady","status":"True","lastProbeTime":null,"lastTransitionTime":null},{"type":"PodScheduled","status":"True","lastProbeTime":null,"lastTransitionTime":"2019-02-20T08:23:15Z"}],"hostIP":"192.168.64.2","podIP":"192.168.64.2","startTime":"2019-02-20T08:23:15Z","containerStatuses":[{"name":"etcd","state":{"running":{"startedAt":"2019-02-20T08:23:16Z"}},"lastState":{},"ready":true,"restartCount":0,"image":"k8s.gcr.io/etcd-amd64:3.2.18","imageID":"docker-pullable://k8s.gcr.io/etcd-amd64@sha256:b960569ade5f37205a033dcdc3191fe99dc95b15c6795a6282859070ec2c6124","containerID":"docker://6a3e4cdc4f6fb308d231e46e15ccefd8d244d37b0303f39feec28d67a4f7f634"}],"qosClass":"BestEffort"}}
I0220 13:25:46.880376    6198 round_trippers.go:386] curl -k -v -XPOST  -H "X-Stream-Protocol-Version: v4.channel.k8s.io" -H "X-Stream-Protocol-Version: v3.channel.k8s.io" -H "X-Stream-Protocol-Version: v2.channel.k8s.io" -H "X-Stream-Protocol-Version: channel.k8s.io" -H "User-Agent: kubectl/v1.11.0 (darwin/amd64) kubernetes/91e7b4f" 'http://0.0.0.0:8444/api/v1/namespaces/kube-system/pods/etcd-minikube/exec?command=%2Fbin%2Fsh&container=etcd&container=etcd&stdin=true&stdout=true&tty=true'
                                                                                        I0220 13:25:51.940498    6198 round_trippers.go:405] POST http://0.0.0.0:8444/api/v1/namespaces/kube-system/pods/etcd-minikube/exec?command=%2Fbin%2Fsh&container=etcd&container=etcd&stdin=true&stdout=true&tty=true 101 Switching Protocols in 5060 milliseconds
                                                                            I0220 13:25:51.940520    6198 round_trippers.go:411] Response Headers:
           I0220 13:25:51.940528    6198 round_trippers.go:414]     Date: Wed, 20 Feb 2019 12:25:51 GMT
                                                                                                       I0220 13:25:51.940534    6198 round_trippers.go:414]     Connection: Upgrade
                                            I0220 13:25:51.940539    6198 round_trippers.go:414]     Upgrade: SPDY/3.1
                                                                                                                      I0220 13:25:51.940544    6198 round_trippers.go:414]     X-Stream-Protocol-Version: v4.channel.k8s.io
/ # %
```

2. Minikube
Kubeconfig:
```

apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUU4RENDQXRpZ0F3SUJBZ0lKQUpBKzlrQzZZZnZlTUEwR0NTcUdTSWIzRFFFQkN3VUFNQmN4RlRBVEJnTlYKQkFNTURDb3VhM2x0WVM1c2IyTmhiREFlRncweE9EQTNNVGd3T0RNNE1UTmFGdzB5T0RBM01UVXdPRE00TVROYQpNQmN4RlRBVEJnTlZCQU1NRENvdWEzbHRZUzVzYjJOaGJEQ0NBaUl3RFFZSktvWklodmNOQVFFQkJRQURnZ0lQCkFEQ0NBZ29DZ2dJQkFPZ05XbVROOU1ranlrUjdvQ0JGSFVqL01EcWh5bml3NEJITGo4ZTdDTFV5dFdwVHZXTkoKU1FiaFZDK3c5NkhHbU50MHZGUTR4OExUa3NNUmorcVZrdkcwKzBDTE1WQm14UjBVdnpZR09QRVRKREtsNTZkcQppMEM1Y2dnU3dkNjcveWxRZmQxc3FEVVBHM0pVZlNOSFFRWSs1SkUwaUpjZXhCQ3cvS200UXlUQXB0aEwzdEgxCm1ZRFBBQ0hUdVpsbGc2RVN4Z2RKY20rVmg5UkRvbWlON250ZjBZVG1xV1VIZXhZUkUrcTFGY0VhUHg5L2Q5QUwKZWd1WXZHTkduMVA4K1F3ZE5DKzVaMEVGa1EyS3RtalRBR09La2xJUG5NQkZubXhGSEtNZzNTa3RkYzYzSU5TNwpjSDM3dDNjb2l4N09HcDllcnRvSFZ5K2U5KzdiYnI3Z0lhWHNORVZUcjRGWXJIMlNPeHI1MjVRQUpHTys1dllQCkNYSlJJdXFNZWU3ZTg3aDRIU3JFVVdWTnhOdElBR1ZNYlRtV2F6aDJsSEFoazJjZVIzUkRaQzB3NGFOd3NRU1UKZE1yMmFCaUFobmFNSG40YnZIZnd1OHFRcjF5aVRUNG00SkltRkZTNDN2bGNETDZmVUJnNnF0U2p6QjF0WTFjTwp2Mnl1QXQzR1lKSENVMFd4R1ZEQk80T0ZncEc3aE5SRHoyRzVFMXQvRWU2VDUxdys2K295eXhib01pK09kVWdNCnBZOGlzcDdBTWhtdlZFOUdtNUhwQkpBRjYvcmE1WUNnNDh1SWtybkdZbm85eVNKcGhLU0JsMFRML01PNVFRaUoKb0hFNnV5ZkJXQjlZZ1NBdDl0MjJqU3FSWkpSdEtrbHB6bFJIRkROWTRwZlRraXlPSlFXYlM4RXJBZ01CQUFHagpQekE5TUFrR0ExVWRFd1FDTUFBd0N3WURWUjBQQkFRREFnWGdNQ01HQTFVZEVRUWNNQnFDQ210NWJXRXViRzlqCllXeUNEQ291YTNsdFlTNXNiMk5oYkRBTkJna3Foa2lHOXcwQkFRc0ZBQU9DQWdFQUlYYTlwenlyQ2dzMTRTOHUKZVFZdkorNEFzUE9uT1RGcExkaVl5UkVyNXdyNmJuMXUvMjZxc2FKckpxbkkyRk16SmdEQVRwZEtjbXRHYjBUOQp3S2wrYUJHcFFKcThrUWJwakVGTHhaWDJzaUNrRG82WittaUcrRjRKMHpKa3BKK0JHMS92eGZKbk0zK1ptdXQ5Ck9RV2ZjYTN3UHlhTWRDbGIyZjQwYlRFaFo5Mk9kcWlQMzFMbDlHWExSZmhaNTNsUzF0QWdvUGZoR25NbFY4b2MKWmxuSUROK25wS0Nma2tXUDJZUjlRLy9pa01tM01YRm9RSFppaVJseVZHSGFKZWRLMmNOQzlUYk4xNDFTaWZHZQo3V2FsQVBNcWNOQ3F3YStnN2RFSmR3ZjlRMklJTml0SjlDUVprT1dUZElYY2VHK2lZWWUrQXpmK1NkaHBocVdPCllFcDF6ek40dXI5U2VxU3NSaU9WY0RzVUFSa1M0clgrb0Vzb2hHL1Q5OTcrSDhjR2gzczl6TE84emtwRXZKSmEKS05QT0N5ODhVeEFOV2RRejFLMXRKVVQ2c3hkd0FEcXRJQnNPemhYVjlybDRRNStlZExlcmZPcUtCbUFRMUY5Swo2L1l0ZlNyY0JpeXZEU24wdFJ3OHJLRFVQU1hFNDFldXArOURNeThLVGl6T0RPTXVMSnR2dkJrTEFpNGNYQjVBCjQxMjBEdHdZQXNyNzNZYVl2SW8rWjV2OGZ4TjF3M3IwYS9KOVhZQlg3S3p1OFl4MnNUNWtWM2dNTHFCTXBaa3gKY29FTjNSandDMmV4VHl6dGc1ak1ZN2U4VFJ4OFFTeUxkK0pBd2t1Tm01NlNkcHFHNTE3cktJYkVMNDZzbkd0UgpCYUVOK01GeXNqdDU3ejhKQXJDMzhBMFN5dTQ9Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
    server: https://apiserver.kyma.local
  name: kyma.local
contexts:
- context:
    cluster: kyma.local
    user: minikube
  name: kyma.local
current-context: kyma.local
kind: Config
preferences: {}
users:
- name: minikube
  user:
    client-certificate: /Users/i355395/.minikube/client.crt
    client-key: /Users/i355395/.minikube/client.key
```

Exec:
```
kubectl exec -n kube-system etcd-minikube /bin/sh -it -v10
I0220 13:27:17.613443    6250 loader.go:359] Config loaded from file cluster-noauth
I0220 13:27:17.614294    6250 loader.go:359] Config loaded from file cluster-noauth
I0220 13:27:17.614945    6250 loader.go:359] Config loaded from file cluster-noauth
I0220 13:27:17.617247    6250 round_trippers.go:386] curl -k -v -XGET  -H "Accept: application/json, */*" -H "User-Agent: kubectl/v1.11.0 (darwin/amd64) kubernetes/91e7b4f" 'https://apiserver.kyma.local/api/v1/namespaces/kube-system/pods/etcd-minikube'
I0220 13:27:17.647153    6250 round_trippers.go:405] GET https://apiserver.kyma.local/api/v1/namespaces/kube-system/pods/etcd-minikube 200 OK in 29 milliseconds
I0220 13:27:17.647185    6250 round_trippers.go:411] Response Headers:
I0220 13:27:17.647194    6250 round_trippers.go:414]     Content-Length: 3530
I0220 13:27:17.647200    6250 round_trippers.go:414]     Content-Type: application/json
I0220 13:27:17.647206    6250 round_trippers.go:414]     Date: Wed, 20 Feb 2019 12:14:14 GMT
I0220 13:27:17.647211    6250 round_trippers.go:414]     X-Envoy-Upstream-Service-Time: 12
I0220 13:27:17.647216    6250 round_trippers.go:414]     Server: envoy
I0220 13:27:17.647272    6250 request.go:897] Response Body: {"kind":"Pod","apiVersion":"v1","metadata":{"name":"etcd-minikube","namespace":"kube-system","selfLink":"/api/v1/namespaces/kube-system/pods/etcd-minikube","uid":"efca9d07-34e8-11e9-979d-2e2f58705dfb","resourceVersion":"505","creationTimestamp":"2019-02-20T08:24:27Z","labels":{"component":"etcd","tier":"control-plane"},"annotations":{"kubernetes.io/config.hash":"8eebc7054eafdda97dee9e915b0e67cf","kubernetes.io/config.mirror":"8eebc7054eafdda97dee9e915b0e67cf","kubernetes.io/config.seen":"2019-02-20T08:23:15.127801274Z","kubernetes.io/config.source":"file","scheduler.alpha.kubernetes.io/critical-pod":""}},"spec":{"volumes":[{"name":"etcd-data","hostPath":{"path":"/data/minikube","type":"DirectoryOrCreate"}},{"name":"etcd-certs","hostPath":{"path":"/var/lib/minikube/certs//etcd","type":"DirectoryOrCreate"}}],"containers":[{"name":"etcd","image":"k8s.gcr.io/etcd-amd64:3.2.18","command":["etcd","--advertise-client-urls=https://127.0.0.1:2379","--cert-file=/var/lib/minikube/certs/etcd/server.crt","--client-cert-auth=true","--data-dir=/data/minikube","--initial-advertise-peer-urls=https://127.0.0.1:2380","--initial-cluster=minikube=https://127.0.0.1:2380","--key-file=/var/lib/minikube/certs/etcd/server.key","--listen-client-urls=https://127.0.0.1:2379","--listen-peer-urls=https://127.0.0.1:2380","--name=minikube","--peer-cert-file=/var/lib/minikube/certs/etcd/peer.crt","--peer-client-cert-auth=true","--peer-key-file=/var/lib/minikube/certs/etcd/peer.key","--peer-trusted-ca-file=/var/lib/minikube/certs/etcd/ca.crt","--snapshot-count=10000","--trusted-ca-file=/var/lib/minikube/certs/etcd/ca.crt"],"resources":{},"volumeMounts":[{"name":"etcd-data","mountPath":"/data/minikube"},{"name":"etcd-certs","mountPath":"/var/lib/minikube/certs//etcd"}],"livenessProbe":{"exec":{"command":["/bin/sh","-ec","ETCDCTL_API=3 etcdctl --endpoints=https://[127.0.0.1]:2379 --cacert=/var/lib/minikube/certs//etcd/ca.crt --cert=/var/lib/minikube/certs//etcd/healthcheck-client.crt --key=/var/lib/minikube/certs//etcd/healthcheck-client.key get foo"]},"initialDelaySeconds":15,"timeoutSeconds":15,"periodSeconds":10,"successThreshold":1,"failureThreshold":8},"terminationMessagePath":"/dev/termination-log","terminationMessagePolicy":"File","imagePullPolicy":"IfNotPresent"}],"restartPolicy":"Always","terminationGracePeriodSeconds":30,"dnsPolicy":"ClusterFirst","nodeName":"minikube","hostNetwork":true,"securityContext":{},"schedulerName":"default-scheduler","tolerations":[{"operator":"Exists","effect":"NoExecute"}],"priorityClassName":"system-cluster-critical","priority":2000000000},"status":{"phase":"Running","conditions":[{"type":"Initialized","status":"True","lastProbeTime":null,"lastTransitionTime":"2019-02-20T08:23:15Z"},{"type":"Ready","status":"True","lastProbeTime":null,"lastTransitionTime":"2019-02-20T08:23:16Z"},{"type":"ContainersReady","status":"True","lastProbeTime":null,"lastTransitionTime":null},{"type":"PodScheduled","status":"True","lastProbeTime":null,"lastTransitionTime":"2019-02-20T08:23:15Z"}],"hostIP":"192.168.64.2","podIP":"192.168.64.2","startTime":"2019-02-20T08:23:15Z","containerStatuses":[{"name":"etcd","state":{"running":{"startedAt":"2019-02-20T08:23:16Z"}},"lastState":{},"ready":true,"restartCount":0,"image":"k8s.gcr.io/etcd-amd64:3.2.18","imageID":"docker-pullable://k8s.gcr.io/etcd-amd64@sha256:b960569ade5f37205a033dcdc3191fe99dc95b15c6795a6282859070ec2c6124","containerID":"docker://6a3e4cdc4f6fb308d231e46e15ccefd8d244d37b0303f39feec28d67a4f7f634"}],"qosClass":"BestEffort"}}
I0220 13:27:17.657420    6250 round_trippers.go:386] curl -k -v -XPOST  -H "X-Stream-Protocol-Version: v4.channel.k8s.io" -H "X-Stream-Protocol-Version: v3.channel.k8s.io" -H "X-Stream-Protocol-Version: v2.channel.k8s.io" -H "X-Stream-Protocol-Version: channel.k8s.io" -H "User-Agent: kubectl/v1.11.0 (darwin/amd64) kubernetes/91e7b4f" 'https://apiserver.kyma.local/api/v1/namespaces/kube-system/pods/etcd-minikube/exec?command=%2Fbin%2Fsh&container=etcd&container=etcd&stdin=true&stdout=true&tty=true'
                                                                                                 I0220 13:27:17.672853    6250 round_trippers.go:405] POST https://apiserver.kyma.local/api/v1/namespaces/kube-system/pods/etcd-minikube/exec?command=%2Fbin%2Fsh&container=etcd&container=etcd&stdin=true&stdout=true&tty=true 403 Forbidden in 15 milliseconds
                                                                                  I0220 13:27:17.672895    6250 round_trippers.go:411] Response Headers:
                 I0220 13:27:17.672915    6250 round_trippers.go:414]     Date: Wed, 20 Feb 2019 12:14:14 GMT
                                                                                                             I0220 13:27:17.672924    6250 round_trippers.go:414]     Server: envoy
                                            I0220 13:27:17.672932    6250 round_trippers.go:414]     Content-Length: 0
                                                                                                                      I0220 13:27:17.673460    6250 helpers.go:201] server response object: [{
  "metadata": {}
}]
F0220 13:27:17.673480    6250 helpers.go:119] Error from server:
```

**Comment by sjanota** [comment]
There is a problem: envoy does not support SPDY, so whenever apiserver-proxy is hidden behind envoy SPDY requests fail.

File: installation/resources/installer.yaml [hunk 1]
```diff
@@ -85,7 +85,7 @@ spec:
       serviceAccountName: kyma-installer
       containers:
       - name: kyma-installer-container
-        image: eu.gcr.io/kyma-project/develop/installer:55bc6038
+        image: sjanota/kyma-installer:latest
```
Comment (sjanota) [inline installation/resources/installer.yaml]: Needs to be changed

File: resources/core/values.yaml [hunk 2]
```diff
@@ -40,8 +40,8 @@ global:
     version: 736988df
     testNamespace: api-controller-tests
   apiserver_proxy:
-    dir: develop/
-    version: 6e0a7fe2
+    dir: pr/
+    version: PR-2905
```
Comment (sjanota) [inline resources/core/values.yaml]: Needs to be changed

**Comment by sjanota** [comment]
Tests:
 - [x] cluster with xip enabled is working as expected, tests are passing 
 - [ ] cluster with domain is working as expected, tests are passing 
 - [ ] minikube is working as expected, tests are passing

File: resources/core/charts/apiserver-proxy/templates/tests/test-apiserver-proxy.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    sidecar.istio.io/inject: "false"
+    "helm.sh/hook": test-success
+  labels:
+    "helm-chart-test": "true"
+spec:
+  hostNetwork: true
+  dnsPolicy: ClusterFirstWithHostNet
+  containers:
+    - name: {{ .Release.Name }}-apiserver-proxy-test
+      image: eu.gcr.io/kyma-project/pr/apiserver-proxy-test:PR-2916
```
Comment (kubadz) [inline resources/core/charts/apiserver-proxy/templates/tests/test-apiserver-proxy.yaml]: Remember to move that to values

File: resources/core/charts/apiserver-proxy/templates/tests/test-apiserver-proxy.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  name: {{ .Release.Name }}-apiserver-proxy-test
+  namespace: kyma-system
+  annotations:
+    sidecar.istio.io/inject: "false"
+    "helm.sh/hook": test-success
+  labels:
+    "helm-chart-test": "true"
+spec:
+  hostNetwork: true
+  dnsPolicy: ClusterFirstWithHostNet
```
Comment (kubadz) [inline resources/core/charts/apiserver-proxy/templates/tests/test-apiserver-proxy.yaml]: `hostNetwork` and `dnsPolicy` should be applied only on local installation

File: resources/core/charts/apiserver-proxy/templates/tests/test-apiserver-proxy.yaml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+{{- end }}
+  containers:
+    - name: test-{{ .Release.Name }}-apiserver-proxy
+      image: eu.gcr.io/kyma-project/pr/apiserver-proxy-test:PR-2916
+      imagePullPolicy: Always
+      env:
+      - name: POD_NAME
+        value: test-{{ .Release.Name }}-apiserver-proxy
+      - name: DOMAIN
+        value: kf-test.kyma.pro
```
Comment (kubadz) [inline resources/core/charts/apiserver-proxy/templates/tests/test-apiserver-proxy.yaml]:
```suggestion
        value: {{ .Values.global.domainName }}
```

**Pull request merged**